src/data/sign.lean

```diff
@@ -202,3 +202,30 @@ def sign_hom : α →*₀ sign_type :=
                mul_neg_of_pos_of_neg, mul_pos] }
 
 end linear_ordered_ring
+
+section add_group
+
+variables [add_group α] [preorder α] [decidable_rel ((<) : α → α → Prop)] {a : α}
+
+lemma left.sign_neg_eq_neg_sign [covariant_class α α (+) (<)] : sign (-a) = - sign a :=
```

Comment:
Could probably abbreviate this to `sign_neg`, similarly below

Reply:
Since the `left` and `right` mean that in fact there's no conflict with the `sign_neg` lemma further up the file, I've done that.